qdrant/v1.2.x/quantization.md

```diff
@@ -46,8 +46,19 @@ Please refer to the [Quantization Tips](#quantization-tips) section for more inf
 
 ## Product Quantization
 
-Currently, Work-in-progress.
+*Available in Qdrant since v1.2.0*
```

Comment:
I believe we use this everywhere else:

```suggestion
*Available since v1.2.0*
```

@davidmyriel suggested to use `Available as of v1.2.0`, but I think we should change that everywhere in a separate PR.

Reply:
I propose to do it once in a separate PR

Reply:
ill take care of all of these little details in a doc overhaul